examples/mockups/product.html

```diff
@@ -0,0 +1,466 @@
+              Go to Application
+            </a>
+          </div>
+        </div>
+    
+      </div>
+    </div>
+    <!-- end Top Menu -->
+
+
+    <!-- Contents 
+    ----------------------------------- -->
+
+    <!-- How it works -->
+    <section class="ui vertical segment very padded" id="how-it-works">
+      <div class="ui container center aligned">
+        <h1 class="ui header massive"><<i>missing</i>> Infographic</h1>
+        <p>
+          A very informative and detailed infographic that we still need to draw,<br>explaining the full Read the Docs workings, dependencies and features.
+        </p>
+      </div>
+    </section>
+
+    <!-- User Segments -->
+    <section class="ui vertical segment inverted padded" id="user-segments">
+
+      <div class="ui container">
+        <div class="ui grid stackable three column">
+
+          <div class="column">
+
+            <div>
+
+              <h3 class="ui header"><<i>missing</i>> Illustration</h3>
+              <h3 class="ui header">Docs as Code</h3>
+              <p>
+                Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx and Mkdocs, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
+              </p>
+              
+              <div class="ui relaxed divided list">
+
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Issue Tracking</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Version Control</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Plain Text Markup</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Code Reviews</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Automated Tests</div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">About the engineering workflow</a>
```

Comment:
> Though, what do you think about matching the button style with the icons?

It's more colorful. It can work. I changed it for now to test but I think this is another thing we'll probably want to come back and review after the content is final, in this case, especially the 3 images and infographic above as we'll probably need to balance the splash of color here with the one above.

Reply:
I also tweaked the green a little to one a bit less electric. Probably would not go as dark as yours to keep it balanced with the brand blue. What do you think of this one? I think it works both on light and on dark backgrounds.

![Screenshot 2021-12-03 091404](https://user-images.githubusercontent.com/4049894/144576994-f00e29b9-8759-428c-ae65-5fcb0814f0fb.png)

Reply:
> this is another thing we'll probably want to come back and review after the content is final

Yeah, absolutely. I wasn't sure about having the three buttons be a pretty prominent green. It might depend on how badly we want the user to go to the child pages as well, if it's a major push then more prominent styles might help us. Agreed we can tweak after some content, I'm not particular about this change